<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" retried="1" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="63680" started-at="2022-01-28T13:23:51 IST" finished-at="2022-01-28T13:24:54 IST">
    <groups>
      <group name="Regression">
        <method signature="COM_LoginFunctionality.TC04_COM_LoginFunctionality()[pri:0, instance:com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality@57d5872c]" name="TC04_COM_LoginFunctionality" class="com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality"/>
      </group> <!-- Regression -->
    </groups>
    <test name="Default test" duration-ms="63680" started-at="2022-01-28T13:23:51 IST" finished-at="2022-01-28T13:24:54 IST">
      <class name="com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality">
        <test-method status="PASS" signature="initialSetup()[pri:0, instance:com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality@57d5872c]" name="initialSetup" is-config="true" duration-ms="3780" started-at="2022-01-28T13:23:47 IST" finished-at="2022-01-28T13:23:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialSetup -->
        <test-method status="PASS" signature="initiate()[pri:0, instance:com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality@57d5872c]" name="initiate" is-config="true" duration-ms="136" started-at="2022-01-28T13:23:51 IST" finished-at="2022-01-28T13:23:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initiate -->
        <test-method status="PASS" signature="driverInitialization()[pri:0, instance:com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality@57d5872c]" name="driverInitialization" is-config="true" duration-ms="26388" started-at="2022-01-28T13:23:51 IST" finished-at="2022-01-28T13:24:18 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverInitialization -->
        <test-method status="SKIP" signature="TC04_COM_LoginFunctionality()[pri:0, instance:com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality@57d5872c]" name="TC04_COM_LoginFunctionality" duration-ms="13467" retried="true" started-at="2022-01-28T13:24:18 IST" finished-at="2022-01-28T13:24:31 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Invalid Credentials passed. Please try again...] but found [The combination of user name and password you entered does not match any existing accounts. Please try again.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Invalid Credentials passed. Please try again...] but found [The combination of user name and password you entered does not match any existing accounts. Please try again.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.perfaware.automation.oms.sterling.common.com.pageobjects.Page_COM_Login.captureLoginError(Page_COM_Login.java:71)
at com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality.TC04_COM_LoginFunctionality(COM_LoginFunctionality.java:91)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC04_COM_LoginFunctionality -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality@57d5872c]" name="tearDown" is-config="true" duration-ms="0" started-at="2022-01-28T13:24:31 IST" finished-at="2022-01-28T13:24:31 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="driverInitialization()[pri:0, instance:com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality@57d5872c]" name="driverInitialization" is-config="true" duration-ms="10489" started-at="2022-01-28T13:24:31 IST" finished-at="2022-01-28T13:24:41 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverInitialization -->
        <test-method status="FAIL" signature="TC04_COM_LoginFunctionality()[pri:0, instance:com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality@57d5872c]" name="TC04_COM_LoginFunctionality" duration-ms="12479" started-at="2022-01-28T13:24:41 IST" finished-at="2022-01-28T13:24:54 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Invalid Credentials passed. Please try again...] but found [The combination of user name and password you entered does not match any existing accounts. Please try again.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Invalid Credentials passed. Please try again...] but found [The combination of user name and password you entered does not match any existing accounts. Please try again.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.perfaware.automation.oms.sterling.common.com.pageobjects.Page_COM_Login.captureLoginError(Page_COM_Login.java:71)
at com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality.TC04_COM_LoginFunctionality(COM_LoginFunctionality.java:91)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC04_COM_LoginFunctionality -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality@57d5872c]" name="tearDown" is-config="true" duration-ms="0" started-at="2022-01-28T13:24:54 IST" finished-at="2022-01-28T13:24:54 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="sendReports()[pri:0, instance:com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality@57d5872c]" name="sendReports" is-config="true" duration-ms="9127" started-at="2022-01-28T13:24:55 IST" finished-at="2022-01-28T13:25:04 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sendReports -->
      </class> <!-- com.perfaware.automation.oms.sterling.common.testcases.COM_LoginFunctionality -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
